tests/pipelines/stable_diffusion/test_stable_diffusion_sag.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+# Copyright 2022 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+import unittest
+
+import numpy as np
+import torch
+
+from diffusers import StableDiffusionSAGPipeline
+from diffusers.utils import slow, torch_device
+from diffusers.utils.testing_utils import require_torch_gpu
+
+
+torch.backends.cuda.matmul.allow_tf32 = False
+
+
```

Comment:
Could we add a fast tests class using the `PipelineTesterMixin`?

The stable diffusion pipeline tests are a good example to follow
https://github.com/huggingface/diffusers/blob/1e7f96544203724652707dc93a7671c90efd3eeb/tests/pipelines/stable_diffusion/test_stable_diffusion.py#L48 

The stable diffusion pipeline tests also have a few additional tests that aren't a part of the `PipelineTesterMixin` that you shouldn't worry about including. The important part is getting the tests that are in the mixin class to run.

I.e. don't worry about the extra scheduler tests, prompt tests, etc...
https://github.com/huggingface/diffusers/blob/1e7f96544203724652707dc93a7671c90efd3eeb/tests/pipelines/stable_diffusion/test_stable_diffusion.py#L119
https://github.com/huggingface/diffusers/blob/1e7f96544203724652707dc93a7671c90efd3eeb/tests/pipelines/stable_diffusion/test_stable_diffusion.py#L138
https://github.com/huggingface/diffusers/blob/1e7f96544203724652707dc93a7671c90efd3eeb/tests/pipelines/stable_diffusion/test_stable_diffusion.py#L172

Reply:
^ Should note, I'm happy to help on adding these if you would like :)

Reply:
I added the Mixin class with a proper input, but am wondering what else should be added. Would you help me with this?

Reply:
The pipeline mixin adds the tests, so already good :)